thetis/utility.py

```diff
@@ -1727,8 +1727,30 @@ def move_to_nearest_cell_center(location):
 
 
 class DepthExpression:
-    """
+    r"""
     Construct expression for depth depending on options
+
+
+    If `not use_nonlinear_equations`, then the depth is simply the bathymetry:
+        :math:`H = h`
+    Otherwise we include the free surface elevation:
+        :math:`H = h + \eta`
+    and if `use_wetting_and_drying`, includes a bathymetry displacement term
+    to ensure a positive depth (see Karna et al. 2011):
+        :math:`H = h + f(h+\eta) + \eta`
+    where
+        :math:`f(h+\eta) = (\sqrt{(h+\eta)^2 +\alpha^2} - (h+\eta))/2
+    This introduces a wetting-drying parameter :math:`\alpha`, with dimensions
+    of length. The value for :math:`\alpha` is specified by
+    `wetting_and_drying_alpha`, in units of meters. The default value is 0.5,
+    but the appropriate value is problem specific and should be set by the user.
+
+    An approximate method for selecting a suitable value for :math:`\alpha` is suggested
```

Comment:
This bit is already in the doc of `shallowwater_eq.py`. Maybe no need to duplicate here?